bitnami/airflow/templates/_helpers.tpl

```diff
@@ -222,6 +222,16 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return true if a SQL connection string is used to connect to the database
+*/}}
+{{- define "airflow.database.useSqlConnection" -}}
+{{- if and (not .Values.postgresql.enabled) (or .Values.externalDatabase.sqlConnection (and .Values.externalDatabase.existingSecret .Values.externalDatabase.existingSecretSqlConnectionKey)) -}}
+    {{- true -}}
+{{- else -}}
```

Comment:
Is this `else` really needed?

Reply:
good catch, done at https://github.com/bitnami/charts/pull/30417/commits/23960a99b1037ae30d3122baa7b4def9d6cb6d11